src/internal/clusterstate/v2.5.0/collection.go

```diff
@@ -280,6 +286,8 @@ func migratePostgreSQLCollection(ctx context.Context, tx *pachsql.Tx, name strin
 				query := fmt.Sprintf("update collections.%s set %s where key = :oldKey", col.table, strings.Join(updateFields, ", "))
 
 				_, err = col.tx.NamedExecContext(ctx, query, params)
```

Comment:
should we check this error since we have `i++`? or is that what `col.mapSQLError()` is for?

Reply:
That’s what the `col.mapSQLError()` is for, as you guessed — it’s from my merge of master into this branch.